server/src/frontend_api.rs

```diff
@@ -1346,7 +1510,7 @@ mod tests {
             .set_json(json!({ "companyId": "bricks"}))
             .to_request();
         let result: FrontendResult = test::call_and_read_body_json(&app, req).await;
-        assert!(result.toggles.is_empty());
+        assert_eq!(result.toggles.len(), 1);
```

Comment:
Turns out we had a test that specifically checked that top-level properties don't work before. I've changed the assertion for now, but we could maybe also remove the last request and assertion?

OR we could treat this as verification for the post properties and remove the post test earlier in the file?

Reply:
I'd say the earlier test is enough, feel free to remove this one.